package.json

```diff
@@ -1,51 +1,70 @@
 {
-  "name": "duplicate-package-checker-webpack-plugin",
-  "version": "2.1.0",
-  "description":
-    "Webpack plugin that warns you when multiple versions of the same package exist in a build.",
-  "main": "./lib/index.js",
-  "scripts": {
-    "test": "jest test/**/*.test.js --runInBand",
-    "build": "babel src --out-dir lib",
-    "prepublish": "npm run build",
-    "precommit": "lint-staged",
-    "pretty": "prettier --write \"src/**/*.{json,js}\" \"test/**/*.{json,js}\""
+  "name": "@cerner/duplicate-package-checker-webpack-plugin",
+  "version": "1.0.0",
+  "description": "Webpack plugin that warns you when multiple versions of the same package exist in a build.",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=8.9.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/duplicate-package-checker-webpack-plugin.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "duplicate-package-checker-webpack-plugin"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/duplicate-package-checker-webpack-plugin/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
   },
-  "lint-staged": {
-    "*.{js,json}": ["prettier --write", "git add"]
+  "scripts": {
+    "clean": "rm package-lock.json; rm -rf node_modules",
+    "clean:install": "npm run clean && npm install",
+    "compile": "babel src --out-dir lib --copy-files",
```

Comment:
what are we using that needs to be transpiled?

Reply:
Nothing if we're running this at build time